mobile/pubspec.yaml

```diff
@@ -173,8 +177,8 @@ dependencies:
       ref: 2cea396
   screenshot: ^3.0.0
   scrollable_positioned_list: ^0.3.5
-  sentry: ^8.13.2
-  sentry_flutter: ^8.13.2
+  sentry: ^8.14.1
+  sentry_flutter: ^8.14.1
```

Comment:
Best to not modify this dependency within this PR?

Reply:
Another PR created.